### DISCUSSION TITLE
about iTwin.js 3 backgroundMap

### DISCUSSION THREAD

**Discussion by shenshengyi** [iTwin/itwinjs-core#3048] [Q&A] [answered]
hi, how does iTwin.js 3.0.0-dev.185 version open the background map correctly? Thanks.

According to the following 2 methods, it cannot be opened correctly (it is possible in iTwin.js 2 version, the default bing map). 
1.
![view](https://user-images.githubusercontent.com/26344907/149700975-354e42f3-731b-4bdc-8636-57f9328033dc.png)
2.
 vp.viewFlags = vp.viewFlags.with("backgroundMap", true);

**Comment by aruniverse** [comment, marked as answer]
Take a look at this breaking change, https://github.com/iTwin/itwinjs-core/blob/release/3.0.x/docs/changehistory/3.0.0.md#removed-default-api-keys. You will need to provide your own api keys for background maps.

**Reply by shenshengyi** [reply]
OK,Thanks.
